cfme/tests/configure/test_logs.py

```diff
@@ -108,12 +109,6 @@ def test_provider_log_updated(appliance, provider, log_exists):
     assert log_before != log_after, "Log hashes are the same"
 
 
-@pytest.mark.meta(blockers=[BZ(1633656,
-                               unblock=lambda provider: provider.one_of(AzureProvider, EC2Provider),
-                               forced_streams=["5.10", "upstream"]
-                               )
-                            ]
-                  )
 def test_provider_log_level(appliance, provider, log_exists):
```

Comment:
Thoughts on parametrizing this test against log level?

Reply:
not a good idea considering why I'm rewriting this test :) originally I was trying to see DEBUG output but it wasn't shown for some providers, which turns out to be not a bug https://bugzilla.redhat.com/show_bug.cgi?id=1633656